### DISCUSSION TITLE
Chrome Web Push Notifications (Push Action Buttons)

### DISCUSSION THREAD

**Discussion by Monochromish** [ciderapp/Cider#878] [Feature Request]
Would it be possible for Cider to have playback controls on the notifications. An example for Push actions button notifications is given below:
![example](https://user-images.githubusercontent.com/79590499/163592411-c9a63b7d-ca56-443b-a355-bf2144a2e3a2.png)
If i remember correctly, something like [Electrolytic](https://electrolytic.app) can be used to do so.

Thank you!

**Comment by cryptofyre** [comment]
Looks interesting! We’ll be sure to check it out. 

@coredev-uk @vapormusic opinions?

**Reply by coredev-uk** [reply]
I think [this](https://github.com/felixrieseberg/electron-windows-notifications) will probably give us these capabilities.
